<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewWidget</class>
 <widget class="QFrame" name="PreviewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
background-color: rgb(0, 0, 0);
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="ImageLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>230</width>
     <height>230</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>230</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>230</width>
     <height>230</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font: 16px &quot;Microsoft YaHei&quot;;
color: #B4C4D9;
background: transparent;
border:none;
}
QToolTip
{
border:1px solid rgb(118, 118, 118);
background-color: #1f2428;
color:#ffffff;
font-size:14px;
border-radius:4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="SelectCheckBox">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox
{
font: 16px &quot;Microsoft YaHei&quot;;
color: #B4C4D9;
background: transparent;
}
QCheckBox::indicator
{
width: 42px;
height: 42px;
}
QCheckBox::indicator:unchecked
{
image: url(:/CArmWorkStation/ImageSource/Data/uncheck.png);
}

QCheckBox::indicator:checked
{
image: url(:/CArmWorkStation/ImageSource/Data/checked.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ImageNumberLabel">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>11</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font: 16px &quot;Microsoft YaHei&quot;;
color: #FFFFFF;
border:none;
background:transparent;
}
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>188</y>
     <width>70</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>32</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 14px &quot;Microsoft YaHei&quot;;
color: rgb(255, 255, 255);
background:rgba(25,25,25,0.56);
border-radius:2px;
border:1px solid rgba(255,255,255,0.6);
}

QPushButton:hover
{
background:rgba(25,25,25,0.7);
}

QPushButton:pressed
{
background:rgba(0,0,0,0.5);
}

</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="DeleteRemindLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>166</y>
     <width>230</width>
     <height>14</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>230</width>
     <height>14</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font: 14px &quot;Microsoft YaHei&quot;;
color: #FFFFFF;
background: transparent;
border:none;
}
</string>
   </property>
   <property name="text">
    <string>71小时91分后删除</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="FoldCheckBox">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>183</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>42</width>
     <height>45</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox
{
font: 16px &quot;Microsoft YaHei&quot;;
color: #B4C4D9;
background: transparent;
}
QCheckBox::indicator
{
width: 42px;
height: 42px;
}
QCheckBox::indicator:unchecked
{
image: url(:/CArmWorkStation/ImageSource/Data/fold.png);
}

QCheckBox::indicator:checked
{
image: url(:/CArmWorkStation/ImageSource/Data/unfold.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="CustomQLabel" name="ImageCountLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>198</y>
     <width>18</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>48</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font: 16px &quot;Microsoft YaHei&quot;;
color: rgb(0, 0, 0);
background: transparent;
border:none;
}
</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQLabel</class>
   <extends>QLabel</extends>
   <header>CustomQLabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../CArmWorkStation.qrc"/>
 </resources>
 <connections/>
</ui>
